pages/index.tsx

```diff
@@ -106,7 +106,7 @@ const Home: NextPage = () => {
 						<CardBody>
 							<VStack gap="25px" alignItems={"flex-start"}>
 								<Button
-									isDisabled={!isReadyToUse}
+									isDisabled // ={!isReadyToUse}
```

Comment:
why this?

Reply:
I reverted this extra commit. Please review with just the link updates. 🙏